webapp/app/data_access/form_data_controller.py

```diff
@@ -0,0 +1,35 @@
+import redis
+import redislite
+from app.config import Config
+from pydantic import MissingError
+
+redis_url = Config.SESSION_DATA_STORAGE_URL
+if redis_url == 'None':
```

Comment:
Personally, I would make that more explicit by setting a config value USE_REDIS_LITE and check for that.

Reply:
Jep you're right, it is cleaner that way. I will change it